CMakeLists.txt

```diff
@@ -418,13 +419,24 @@ if(VCPKG_PDB_SUFFIX)
 endif()
 
 # === Target: generate-message-map ===
+if (VCPKG_ARTIFACTS_DEVELOPMENT)
+    add_custom_target(generate-message-map
+       COMMAND vcpkg x-generate-default-message-map locales/messages.json ${CMAKE_CURRENT_LIST_DIR}/ce/ce/locales/messages.json
+       DEPENDS ${CMAKE_CURRENT_LIST_DIR}/ce/ce/locales/messages.json
+       WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR}
+       COMMENT "Update locales/messages.json"
+       VERBATIM
+    )
```

Comment:
Huh why is this behind `VCPKG_ARTIFACTS_DEVELOPMENT`?

Reply:
You still need the old version when you do "normal" vcpkg-tool development

Reply:
> Huh why is this behind `VCPKG_ARTIFACTS_DEVELOPMENT`?

Because `VCPKG_ARTIFACTS_DEVELOPMENT` implies that you have node / NPM / rush et al. set up and working on your machine. People deploying vcpkg on a platform we don't have a binary release for can't / shouldn't be expected to provide those dependencies.

Reply:
Yeah but only executing the command `vcpkg x-generate-default-message-map locales/messages.json` is needed for vcpkg-tool development to make the ci happy. So that should be added as target if `VCPKG_ARTIFACTS_DEVELOPMENT` is false.

Reply:
> Yeah but only executing the command `vcpkg x-generate-default-message-map locales/messages.json` is needed for vcpkg-tool development to make the ci happy.

That command needs `VCPKG_ARTIFACTS_DEVELOPMENT` to be on after this change.

Reply:
I have created https://github.com/microsoft/vcpkg-tool/pull/854